recipes-kernel/linux/files/patches-6.1/0078-arm64-dts-ti-iot2050-Disable-R5-lockstep-for-all-PG2.patch

```diff
@@ -0,0 +1,77 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Baocheng Su <baocheng.su@siemens.com>
+Date: Fri, 1 Dec 2023 19:25:47 +0800
+Subject: [PATCH] arm64: dts: ti: iot2050: Disable R5 lockstep for all PG2
+ boards
+
+The R5 lockstep disabling should be common for all PG2 boards, move it
+from variants dts to common-pg2.dtsi.
+
+Signed-off-by: Baocheng Su <baocheng.su@siemens.com>
+---
+ arch/arm64/boot/dts/ti/k3-am65-iot2050-common-pg2.dtsi    | 7 ++++++-
+ arch/arm64/boot/dts/ti/k3-am6548-iot2050-advanced-m2.dts  | 5 -----
+ arch/arm64/boot/dts/ti/k3-am6548-iot2050-advanced-pg2.dts | 7 +------
+ 3 files changed, 7 insertions(+), 12 deletions(-)
+
+diff --git a/arch/arm64/boot/dts/ti/k3-am65-iot2050-common-pg2.dtsi b/arch/arm64/boot/dts/ti/k3-am65-iot2050-common-pg2.dtsi
+index e9b57b87e42e..42adb8815f38 100644
+--- a/arch/arm64/boot/dts/ti/k3-am65-iot2050-common-pg2.dtsi
++++ b/arch/arm64/boot/dts/ti/k3-am65-iot2050-common-pg2.dtsi
+@@ -1,6 +1,6 @@
+ // SPDX-License-Identifier: GPL-2.0
+ /*
+- * Copyright (c) Siemens AG, 2021
++ * Copyright (c) Siemens AG, 2021-2023
+  *
+  * Authors:
+  *   Chao Zeng <chao.zeng@siemens.com>
+@@ -9,6 +9,11 @@
+  * Common bits of the IOT2050 Basic and Advanced variants, PG2
+  */
+ 
++&mcu_r5fss0 {
++	/* lock-step mode not supported on PG2 boards */
++	ti,cluster-mode = <0>;
++};
++
```

Comment:
Likely not critical, but now the basic pg2 will pull this also via `k3-am6528-iot2050-basic-common.dtsi`. Maybe also move the related bits from `k3-am6528-iot2050-basic-common.dtsi` to `k3-am6528-iot2050-basic.dts`?

Reply:
https://github.com/siemens/linux/commit/350bbd49605c4fd26bd4c40e212cf4c800f73367